damnit/ctxsupport/ctxrunner.py

```diff
@@ -448,12 +442,18 @@ def save_hdf5(self, hdf5_path, reduced_only=False):
 
                 if isinstance(obj, str):
                     f.create_dataset(path, shape=(), dtype=h5py.string_dtype())
+                elif isinstance(obj, PNGData):  # Thumbnail
+                    f.create_dataset(path, shape=len(obj.data), dtype=np.uint8)
+                    f[path].attrs['damnit_png'] = 1
```

Comment:
I don't have an explicit summary type for PNG, because my idea is to recognise them from the data (i.e. an internal tag rather than an external one). So I've now changed it to follow the same pattern in the `.reduced` section of the HDF5 files - it looks for a 1D dataset of uint8 that starts with the PNG signature.